<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomMainWindow</class>
 <widget class="QMainWindow" name="RoomMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="RoomPlaypushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="RoomCominglabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正黄草简体</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>  新生入学中......</string>
    </property>
   </widget>
   <widget class="QLabel" name="RoomRulelabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>云书法手书罗西硬笔行楷</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>  新生交流群</string>
    </property>
   </widget>
   <widget class="QLabel" name="RoomLoadinglabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="RoomPassageWaylabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>567</width>
      <height>426</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>291</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="senderLine">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <zorder>RoomPassageWaylabel</zorder>
   <zorder>RoomPlaypushButton</zorder>
   <zorder>RoomRulelabel</zorder>
   <zorder>RoomCominglabel</zorder>
   <zorder>pushButton</zorder>
   <zorder>senderLine</zorder>
   <zorder>textBrowser</zorder>
   <zorder>RoomLoadinglabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>567</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRoom">
    <property name="title">
     <string>Room</string>
    </property>
   </widget>
   <addaction name="menuRoom"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
